cip.go

```diff
@@ -286,6 +286,10 @@ func main() {
 
 		if *severityThresholdPtr >= 0 {
 			klog.Info("********** START (VULN CHECK) **********")
+			klog.Info("DISCLAIMER: Vulnerabilities are found as issues with " +
+				"package binaries within image dependencies, not necessarily " +
+				"with the image dependencies themselves. So a \"fixable\" " +
```

Comment:
"For example, even though a fixed version of the binary is available, it does not necessarily mean that a new version of the image layer is available."

Reply:
Done